tests/bindings/hlsl-to-vulkan-combined.hlsl

```diff
@@ -0,0 +1,9 @@
+//TEST:REFLECTION:-target glsl -profile ps_4_0 -entry main -fvk-t-shift 5 all -fvk-t-shift 7 2  -fvk-s-shift -3 0 -fvk-b-shift 1 2
```

Comment:
Reflection test covers the reflection API, but we still need to ensure that the generated glsl contains the correct bindings.

Add another 
```
//TEST:SIMPLE(filecheck=CHECK): -target glsl -profile ps_4_0 -entry main -fvk-t-shift 5 all -fvk-t-shift 7 2  -fvk-s-shift -3 0 -fvk-b-shift 1 2
```
line here, and we can check for the glsl bindings with lines like:
```
// CHECK: layout(binding = xxx)
// CHECK: {{.*}} sampler
...
```

Reply:
ok will add as a new PR.